protocols/kad/src/lib.rs

```diff
@@ -33,9 +33,6 @@
 //! existing nodes in the kademlia network cannot obtain the listen addresses
 //! of nodes querying them, and thus will not be able to add them to their routing table.
 
-// TODO: we allow dead_code for now because this library contains a lot of unused code that will
-//       be useful later for record store
```

Comment:
It's been 5 years, I don't think we should keep it any longer :upside_down_face:

Reply:
Haha, you never know :P